doc/man3/UI_new.pod

```diff
@@ -64,10 +64,7 @@ UI_set_method, UI_OpenSSL, UI_null - user interface
 =head1 DESCRIPTION
 
 UI stands for User Interface, and is general purpose set of routines to
-prompt the user for text-based information.  Through user-written methods
-(see L<ui_create(3)>), prompting can be done in any way
-imaginable, be it plain text prompting, through dialog boxes or from a
-cell phone.
+prompt the user for text-based information.
```

Comment:
that function isn't documented either.  when the page is written the links can be added back.